src/Aspire.Hosting.Nats/PublicAPI.Shipped.txt

```diff
@@ -4,7 +4,6 @@ Aspire.Hosting.ApplicationModel.NatsServerResource.ConnectionStringExpression.ge
 Aspire.Hosting.ApplicationModel.NatsServerResource.NatsServerResource(string! name) -> void
 Aspire.Hosting.ApplicationModel.NatsServerResource.PrimaryEndpoint.get -> Aspire.Hosting.ApplicationModel.EndpointReference!
 Aspire.Hosting.NatsBuilderExtensions
-static Aspire.Hosting.NatsBuilderExtensions.AddNats(this Aspire.Hosting.IDistributedApplicationBuilder! builder, string! name, int? port = null) -> Aspire.Hosting.ApplicationModel.IResourceBuilder<Aspire.Hosting.ApplicationModel.NatsServerResource!>!
```

Comment:
We shouldn't be / can't be making breaking changes. You can only add new API. See https://github.com/dotnet/aspire/pull/5788 for how this can be done.